src/components/application_manager/rpc_plugins/sdl_rpc_plugin/test/commands/mobile/add_command_request_test.cc

```diff
@@ -258,6 +258,26 @@ TEST_F(AddCommandRequestTest, Run_ImageVerificationFailed_EXPECT_INVALID_DATA) {
   request_ptr->Run();
 }
 
+TEST_F(AddCommandRequestTest, Run_ImageVerificationFailed_EXPECT_WARNINGS) {
+  CreateBasicParamsUIRequest();
+  SmartObject& msg_params = (*msg_)[strings::msg_params];
+  SmartObject& image = msg_params[cmd_icon];
+  EXPECT_CALL(mock_message_helper_, VerifyImage(image, _, _))
+      .WillOnce(Return(mobile_apis::Result::WARNINGS));
+
+  am::CommandsMap commands_map;
+  EXPECT_CALL(*mock_app_, commands_map())
+      .WillRepeatedly(Return(DataAccessor<application_manager::CommandsMap>(
+          commands_map, lock_ptr_)));
+  EXPECT_CALL(
+      mock_rpc_service_,
+      ManageHMICommand(HMIResultCodeIs(hmi_apis::FunctionID::UI_AddCommand)))
+      .WillOnce(Return(true));
+  utils::SharedPtr<AddCommandRequest> request_ptr =
```

Comment:
use `std::shared_ptr`, `utils::SharedPtr` no longer exists in develop

Reply:
@jacobkeeler I've rebased this feature branch on the latest develop to use shared pointers from STL.
Otherwise it would require a lot of replaces of `SharedPtr` in the code, but it is an overhead to do that.

Reply:
@jacobkeeler after rebase all `SharedPtr` were replaced with shared pointers from STL